tutor/src/screens/assignments/preview-tooltip.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { OverlayTrigger, Button, Popover } from 'react-bootstrap';
+import { observer, styled } from 'vendor';
+import { TemplateBody } from '../grading-templates/card';
+
+const StyledButton = styled(Button)`
+  &.btn {
+    font-size: 1.6rem;
+  }
+
+  margin-left: 0.8rem;
+`;
+
+const StyledContent = styled(Popover.Content)`
+  .card {
+    border: 0;
+  }
+`;
```

Comment:
This might look a little strange, why not just override TemplateBody, but I ran into styled-components specificity issues.